lib/src/query_service.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:js';
+
+import 'gapi.dart';
+import 'json_js.dart';
+import 'insertion_order_parser.dart';
+import 'proto/insertion_order.pb.dart';
+import 'dv360_query_builder.dart';
+
+class QueryService {
+  QueryService._private();
+
+  static final QueryService _singleton = QueryService._private();
+
+  factory QueryService() {
+    return _singleton;
+  }
+
+  final queryBuilder = DV360QueryBuilder();
+  final _table = <InsertionOrder>[];
```

Comment:
nit: why _table? consider another name or adding more descriptors here

Reply:
Can delete this _table here, not really necessary, and I was coding ahead.